cli/src/cmd/forge/inspect.rs

```diff
@@ -122,14 +132,23 @@ impl Cmd for InspectArgs {
             ContractArtifactFields::StorageLayout => {
                 if pretty {
                     if let Some(storage_layout) = &artifact.storage_layout {
+                        let provider = get_http_provider(
```

Comment:
this will now panic if no url provided and no local node is running, 

I think this should be optional, and only used if set?

Reply:
Fixed.